src/AstGen.zig

```diff
@@ -1451,10 +1455,13 @@ pub fn comptimeExpr(
     rl: ResultLoc,
     node: ast.Node.Index,
 ) InnerError!Zir.Inst.Ref {
-    const prev_force_comptime = gz.force_comptime;
+    const astgen = gz.astgen;
+    if (gz.force_comptime) {
+        return astgen.failNode(node, "comptime here happens inside a comptime scope; it is redundant", .{});
```

Comment:
```suggestion
        return astgen.failNode(node, "redundant comptime keyword in implicitly comptime scope", .{});
```

Error message wording principles:
 * Efficiency of words
 * Precise words